projects/SelfTest/UsageTests/Matchers.tests.cpp

```diff
@@ -242,12 +242,14 @@ namespace { namespace MatchersTests {
 
             ~CustomAllocator() = default;
 
+#if defined(_MSC_VER) && _MSVC_LANG < 201703L
             using std::allocator<T>::address;
-            using std::allocator<T>::allocate;
             using std::allocator<T>::construct;
-            using std::allocator<T>::deallocate;
             using std::allocator<T>::max_size;
             using std::allocator<T>::destroy;
```

Comment:
The deprecated ones should be safe to just delete afaik.

Reply:
Hi horenmar,

Thank you so much for your review! I initially deleted the deprecated methods, and this worked for GCC and Clang, but deleting those methods broke the build on older versions of MSVC (MSVC 2015). That's why I added a ifdef specifically to check the version of MSVC being used to compile it.

Reply:
Right. Yeah, then the fix is correct.

For v3 I deleted the deprecated usings, but also v3 no longer supports VS 2015.